debian/control

```diff
@@ -15,7 +15,7 @@ Build-Depends:
 Vcs-Git: https://github.com/Nicotine-Plus/nicotine-plus.git
 Vcs-browser: https://github.com/Nicotine-Plus/nicotine-plus
 X-Python-Version: >= 3.5
-Homepage: https://github.com/Nicotine-Plus/nicotine-plus
+Homepage: https://nicotine-plus.org
```

Comment:
Maybe I missed it, but is old domain configured to load our markdown now?

Reply:
It will be soon: https://github.com/Nicotine-Plus/nicotine-plus/issues/109#issuecomment-633291564